app/controllers/items_controller.rb

```diff
@@ -693,16 +692,16 @@ def save_and_reindex
   # Permissions
 
   # check that the user can carry out this item modification
-  def forbid_modify
-    return true if current_user.is_admin || @object.can_manage_content?(current_user.roles(@apo))
-    render :status => :forbidden, :text => 'forbidden'
-    false
+  def authorize_manage_obj!
+    authorize! :manage_item, @object
   end
 
-  def forbid_view
-    return true if current_user.is_admin || @object.can_view_content?(current_user.roles(@apo))
-    render :status => :forbidden, :text => 'forbidden'
-    false
+  def authorize_view_obj!
+    authorize! :view_content, @object
+  end
```

Comment:
I think I've misunderstood something. In https://github.com/sul-dlss/argo/pull/596#discussion_r54289914, you pointed out an instance where I was using `manage_item` instead of `manage_content`. After switching it to use `manage_content`, it is now parallel to `view_content`.